code/modules/mob/living/carbon/human/species_types/modder.dm

```diff
@@ -0,0 +1,98 @@
+//gene modded humans
+/datum/species/human/modder
+	name = "Modded Human"
+	id = SPECIES_MODDER
+	//say_mod = "meows"
+
+	mutant_bodyparts = list("tail_human" = "Cat", "ears" = "Cat", "wings" = "None")
+
+	mutantears = /obj/item/organ/ears/cat
+	mutant_organs = list(/obj/item/organ/tail/cat)
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
+	//species_language_holder = /datum/language_holder/felinid
+	payday_modifier = 0.75
+	ass_image = 'icons/ass/asscat.png'
+	examine_limb_id = SPECIES_HUMAN
+
+//Curiosity killed the cat's wagging tail.
+/datum/species/human/modder/spec_death(gibbed, mob/living/carbon/human/H)
+	if(H)
+		stop_wagging_tail(H)
+
+/datum/species/human/modder/spec_stun(mob/living/carbon/human/H,amount)
+	if(H)
+		stop_wagging_tail(H)
+	. = ..()
+
+/datum/species/human/modder/can_wag_tail(mob/living/carbon/human/H)
+	return mutant_bodyparts["tail_human"] || mutant_bodyparts["waggingtail_human"]
+
+/datum/species/human/modder/is_wagging_tail(mob/living/carbon/human/H)
+	return mutant_bodyparts["waggingtail_human"]
+
+/datum/species/human/modder/start_wagging_tail(mob/living/carbon/human/H)
+	if(mutant_bodyparts["tail_human"])
+		mutant_bodyparts["waggingtail_human"] = mutant_bodyparts["tail_human"]
+		mutant_bodyparts -= "tail_human"
+	H.update_body()
+
+/datum/species/human/modder/stop_wagging_tail(mob/living/carbon/human/H)
+	if(mutant_bodyparts["waggingtail_human"])
+		mutant_bodyparts["tail_human"] = mutant_bodyparts["waggingtail_human"]
+		mutant_bodyparts -= "waggingtail_human"
+	H.update_body()
+
+/datum/species/human/modder/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(H.dna.features["ears"] != "None")
+			var/obj/item/organ/ears/cat/ears = new
+			ears.Insert(H, drop_if_replaced = FALSE)
+		else
+			mutantears = /obj/item/organ/ears
+		if(H.dna.features["tail_human"] != "None")
+			var/obj/item/organ/tail/cat/tail = new
+			tail.Insert(H, special = TRUE, drop_if_replaced = FALSE)
+		else
+			mutant_organs = list()
+	return ..()
+
+/datum/species/human/modder/prepare_human_for_preview(mob/living/carbon/human/human)
+	human.hairstyle = "Hime Cut"
+	human.hair_color = "#ffcccc" // pink
+	human.update_hair()
+
+	var/obj/item/organ/ears/cat/cat_ears = human.getorgan(/obj/item/organ/ears/cat)
+	if (cat_ears)
+		cat_ears.color = human.hair_color
+		human.update_body()
```

Comment:
The preview dummy isn't rendering properly at the moment. Currently it's just the same as a regular human:
![image](https://user-images.githubusercontent.com/57483089/167494398-27164770-167b-4db1-bd33-72116fd44587.png)

The lack of ears is caused by the default `dna.features["ears"]` value being `"None"`, so in order to get them to always show up on the preview you'll need to add them in here:
```suggestion
/datum/species/human/modder/prepare_human_for_preview(mob/living/carbon/human/human)
	human.hairstyle = "Hime Cut"
	human.hair_color = "#ffcccc" // pink
	human.update_hair()

	var/obj/item/organ/ears/cat/cat_ears = new
	cat_ears.Insert(human, drop_if_replaced = FALSE)
	cat_ears.color = human.hair_color
	human.update_body()
```
*(The baldness is a base TG issue, so until that's fixed it's fine to leave it)*
<hr>

Felinids get around needing to do this thanks to this part in their `on_species_gain()` proc:
https://github.com/pariahstation/Pariah-Station/blob/b7477dad8f9384cfc9e37c88b570e158b1ec097d/code/modules/mob/living/carbon/human/species_types/felinid.dm#L59-L63
When the dummy changes species, it gets the ears and tail added automatically due to `pref_load` being false by default:
https://github.com/pariahstation/Pariah-Station/blob/b7477dad8f9384cfc9e37c88b570e158b1ec097d/code/modules/client/preferences/middleware/species.dm#L21-L23